### PR TITLE
style(collapsible): collapsible animation

### PR DESCRIPTION
**Pull request by Powerplex**
**TASK**: #2224 

### Description, Motivation and Context

See attached ticket. Adding `Collapsible` animation using both opacity and height animation.

### Types of changes
- [x] 🧾 Documentation
- [x] 💄 Styles

### CODE REVIEW

**Review by andresin87 — APPROVED** [review]
🚀

File: packages/utils/tailwind-plugins/animations/index.js [hunk 1]
```diff
@@ -121,6 +121,14 @@ module.exports = plugin.withOptions(
     theme: {
       extend: {
         keyframes: {
+          collapseIn: {
```
Comment (acd02) [inline packages/utils/tailwind-plugins/animations/index.js:124]:
```suggestion
          standaloneCollapseIn: {
```

see 👉 https://github.com/adevinta/spark/wiki/Conventions-for-Adding-Custom-Styles-in-Tailwind#standalone-keyframes

**Review by acd02 — APPROVED** [review]

File: packages/utils/tailwind-plugins/animations/index.js (hunk 1 above)
Comment (Powerplex) [reply packages/utils/tailwind-plugins/animations/index.js:124]: As it will also be re-used by `Accordion` soon, should it still be prefixed ?

Comment (acd02) [reply packages/utils/tailwind-plugins/animations/index.js:124]: Maybe we could name it `standaloneDisclosureIn` / `standaloneDisclosureOut`. Then, we could use that for both the collapsible and accordion elements, wdyt?

Comment (acd02) [reply packages/utils/tailwind-plugins/animations/index.js]: beware, you used  `standaloneCollapseIn` / `standaloneCollapseOut`, and not `standaloneDisclosureIn` / `standaloneDisclosureOut`

**Pull request merged**